bin/node/e2e-tests/src/lib.rs

```diff
@@ -0,0 +1,189 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020-2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! End to end runtime tests
+
+use substrate_test_runner::{Node, ChainInfo, SignatureVerificationOverride};
+use grandpa::GrandpaBlockImport;
+use sc_service::{TFullBackend, TFullClient, Configuration, TaskManager, new_full_parts};
+use node_runtime::{SignedExtra, Runtime, Event};
+use sp_runtime::generic::Era;
+use std::sync::Arc;
+use sp_inherents::InherentDataProviders;
+use sc_consensus_babe::BabeBlockImport;
+use sp_keystore::SyncCryptoStorePtr;
+use sp_keyring::sr25519::Keyring::{Alice, Bob};
+use node_cli::chain_spec::development_config;
+use sp_consensus_babe::AuthorityId;
+use sc_consensus_manual_seal::{ConsensusDataProvider, consensus::babe::BabeConsensusDataProvider};
+use sp_runtime::{traits::IdentifyAccount, MultiSigner};
+
+type BlockImport<B, BE, C, SC> = BabeBlockImport<B, C, GrandpaBlockImport<BE, B, C, SC>>;
+
+sc_executor::native_executor_instance!(
+	pub Executor,
+	node_runtime::api::dispatch,
+	node_runtime::native_version,
+	SignatureVerificationOverride,
+);
+
+/// ChainInfo implementation.
+struct NodeTemplateChainInfo;
+
+impl ChainInfo for NodeTemplateChainInfo {
```

Comment:
technically you're using the "standard node runtime" not the "node template runtime"

Reply:
This is not addressed and a valid point.